packages/webapp/src/containers/ReleaseBadgeHandler/saga.js

```diff
@@ -0,0 +1,69 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import apiConfig from '../../apiConfig';
+import { loginSelector } from '../userFarmSlice';
+import { axios, getHeader } from '../saga';
+import { createAction } from '@reduxjs/toolkit';
+
+const { releaseBadgeUrl } = apiConfig;
+
+export const checkReleaseBadgeVersion = createAction(`checkReleaseBadgeVersionSaga`);
+
+export function* checkReleaseBadgeVersionSaga({ payload }) {
+  const { currentVersion, setShouldShowBadge } = payload;
+
+  const { user_id, farm_id } = yield select(loginSelector);
+  const header = getHeader(user_id, farm_id);
+
+  try {
+    const result = yield call(axios.get, releaseBadgeUrl, header);
+
+    const { app_version } = result.data;
+
+    if (app_version < currentVersion) {
```

Comment:
This is never true for me and seems to be preventing the functionality from working. Might need an else or a different way to compare versions.

Reply:
Really? It was originally written `!==` (same as app version saga) but it works for me either way and this is the comparison we want (in case two devices have different versions loaded) 🤔 When you change your `APP_VERSION` constant in your testing, are you bumping it up alphabetically? Normal semantic versioning (bumping the minor / patch version) should work fine if you're not going into double-digits.

Reply:
It works once I have a record (probably with a number) in the db but it doesn't create one until it works. So its probably stuck in a `null < null` situation. 

Do strings compare well this way? Are we sure future Litefarmers wont go high on patch versions or change version naming to some other approach?

Reply:
nvm this is what I am seeing: 
![Screenshot 2023-12-21 at 4 49 30 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/30075973/65652fd7-ac9d-4fd0-bfdd-5bb8aa4a9475)

Reply:
Ahhhhh okay thank you!! This is because I established my database records with the other method.

I'm not sure about the versioning; the fact that we're up to 3.6 potentially does make me nervous about double digits 🤔 Maybe I'll do a more appropriate semantic versioning compare then. I don't think we will change our whole version scheme (given that we've done so many releases on it already) but maybe a note for future LiteFarmers then!